qfieldsync/gui/map_layer_config_widget.py

```diff
@@ -55,6 +55,8 @@ def supportLayerPropertiesDialog(self):
 
 
 class MapLayerConfigWidget(QgsMapLayerConfigWidget, WidgetUi):
+    PROPERTY_GEOMETRY_LOCKED = 1
```

Comment:
It is not super clear to me what the `1` stands for?

Reply:
@suricactus , with  properties, you need to provide an integer identifier for a given property (see https://github.com/opengisch/qfieldsync/pull/548/files#diff-cb0fe03aac5d043fa6713776d09aaa241751d0e5b4f9d8d7c2ee1b20d58a6905R89), much better to have it as a PROPERTY_SOMETHING constant than an integer of 1 entered in the data-defined button's init() function.

Reply:
I see. Thanks for the clarification!